src/components/inputModal/index.tsx

```diff
@@ -23,9 +22,8 @@ const StudentInputModal: React.FC<IModalProps> = (props) => {
     grades: "",
   };
 
-  const grades = ["A", "B", "C", "D", "F"];
-  const subjects = ["English", "Math", "Science", "History"];
-  const modalType = props.isEdit ? ModalType.EDIT : ModalType.ADD;
+  // const modalType = props?.isEdit ? MODAL_TYPE.EDIT : MODAL_TYPE.ADD;
+  const modalType = MODAL_TYPE.ADD;
```

Comment:
it's a const and already a static string is assigned to it. We can directly use `MODAL_TYPE.ADD`